framework-plugins/lisk-framework-http-api-plugin/src/controllers/peers.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { Request, Response, NextFunction } from 'express';
+import { isUInt32, isString } from '@liskhq/lisk-validator';
+import { BaseChannel } from 'lisk-framework';
+
+export enum PeerState {
+	connected = 'connected',
+	disconnected = 'disconnected',
+}
+
+interface PeerInfo {
```

Comment:
Can you import these types from `lisk-p2p`

Reply:
we are trying not to have a dependency on p2p lib so not importing from there